build/containerize.sh

```diff
@@ -15,7 +15,7 @@ rm -rf $target; mkdir -p $target
 
 ## Generate the manifest of the image to be built
 vessel image --registry docker.io \
-       --repository alangh/tekton-watcher \
+       --repository tekton-watcher \
```

Comment:
Will vessel find the correct repository without `alangh/` in the name ?

Reply:
Vessel will do. There could be issues to deploy the manifest to Kubernetes wich will require modifications outside the scope of documentation.